pkg/drivers/kic/oci/oci.go

```diff
@@ -159,11 +166,23 @@ func HostPortBinding(ociBinary string, ociID string, contPort int) (int, error)
 	if err := PointToHostDockerDaemon(); err != nil {
 		return 0, errors.Wrap(err, "point host docker-daemon")
 	}
-	cmd := exec.Command(ociBinary, "inspect", "-f", fmt.Sprintf("'{{(index (index .NetworkSettings.Ports \"%d/tcp\") 0).HostPort}}'", contPort), ociID)
-	out, err := cmd.CombinedOutput()
-	if err != nil {
-		return 0, errors.Wrapf(err, "getting host-bind port %d for container ID %q, output %s", contPort, ociID, out)
+	var out []byte
+	var err error
+	if ociBinary == Podman {
```

Comment:
These if statements are substantial enough that we should probably have it use separate Podman & Docker helper functions.

Reply:
done